quill-jdbc/src/test/scala/io/getquill/context/jdbc/sqlserver/JdbcContextSpec.scala

```diff
@@ -56,3 +56,9 @@ class JdbcContextSpec extends Spec {
     testContext.run(qr4.filter(_.i == lift(inserted))).head.i mustBe inserted
   }
 }
+
+class PendingUntilFixed extends Spec {
```

Comment:
Does this make the test be ignored? you can also do `"test descriotion" in pendingUntilFixed {`

Reply:
@fwbrasil Nope :). I agree name is confusing, I've added this for 100% coverage, to test SqlServerJdbcContext constructors. 
As you can see I pass "testPostgresDB" to make it work. "PendingUntilFixed" means let's keep it until we have stable sqlserver image.

This is somehow necessary since we have now 100% for each module except `quill-core`. I didn't have much success there, besides test cases like `"not compiled code" must not compile` are not caught up by coverage plugin :(